onnx/common/ir.h

```diff
@@ -785,8 +785,8 @@ class OpSetID final {
     explicit OpSetID(const int64_t version)
       :domain_(""), version_(version) {}
 
-    explicit OpSetID(const std::string& domain, int64_t version)
-      :domain_(domain), version_(version) {}
+    explicit OpSetID(const std::string domain, int64_t version)
```

Comment:
Either const std::string& + copy constructor, or std::string + move constructor. 

Cannot be a const string + move constructor...

Reply:
Should we establish, especially given review from @smessmer, that protocol is typically to use the copy constructor with an lvalue reference?